src/components/HouseCard.jsx

```diff
@@ -49,14 +49,12 @@ const HeaderContainer = styled.div`
 
 const Button = styled.button`
   align-self: flex-end;
-  background-color: ${props => props.active ? 'darkgray' : 'none'};
-  color: ${props => props.active ? 'white' : 'black'};
+  background-color: ${props => props.active ? '#3A3A3A' : '#FFF'};
+  color: ${props => props.active ? '#000' : '#FFF'};
```

Comment:
Do we want `color` of text to be white when unactive?

Reply:
I think I inherited a lot of these changes from your branch... Do we?

Reply:
Switch it up or we can fix it in another PR